src/changes-backwards-compatible.apib

```diff
@@ -10,6 +10,11 @@ We list all backwards-compatible additions here. These are currently available i
 - We added an optional filter on `ids` to `invoices.list`.
 - We added `creditNote.booked` and `product.added` to the available webhooks list.
 
+#### June 2019
```

Comment:
I'm not sure how this fits the order of things here :)

Reply:
I guess we can reevaluate before merging